front/components/assistant/conversation/messages/MessageActions.tsx

```diff
@@ -222,27 +217,21 @@ function EmojiSelector({
   const buttonRef = useRef<HTMLDivElement>(null);
 
   return (
-    <DropdownMenu>
-      <DropdownMenu.Button>
-        <div ref={buttonRef}>
-          <Button
-            variant="tertiary"
-            size="xs"
-            icon={ReactionIcon}
-            labelVisible={false}
-            label="Reaction picker"
-            disabledTooltip
-            type="menu"
-            disabled={disabled}
-          />
-        </div>
-      </DropdownMenu.Button>
-      <DropdownMenu.Items
-        width={350}
-        origin="bottomRight"
-        overflow="visible"
-        variant="no-padding"
-      >
+    <Popover
+      fullWidth={true}
```

Comment:
nit: from @flvndvd standards, shouldn't it be `fullWidth` and that's it ?

Reply:
are you sure it's fullwidth ? (the old one didn't have that)

Reply:
The `Popover` component is a new component, we didn't have one and were misusing the `DropdownMenu`. Basically passing `fullWidth` allow the `Popover` to grow, see [here](https://github.com/dust-tt/dust/blob/f9ed56e2b6d482097c8dfa914193ae21380e00e3/sparkle/src/components/Popover.tsx#L42)